app/models/miq_server/role_management.rb

```diff
@@ -46,6 +46,7 @@ def set_active_role_flags
     self.has_active_userinterface = self.has_active_role?("user_interface")
     self.has_active_websocket     = self.has_active_role?("websocket")
     self.has_active_webservices   = self.has_active_role?("web_services")
+    self.has_active_cockpit_ws    = MiqCockpitWsWorker.can_activate_cockpit?
```

Comment:
> if the server assigned the relevant role doesn't have cockpit-ws and cockpit-ssh installed 

Is that a situation that can happen?  I would expect that cockpit-ws would just be on all of our appliances as part of the build process.  That way *any* server can become a cockpit enabled server, and they are then all generic.